src/api-serverless/src/ci-pipeline-alerts/ci-pipeline-alert.service.ts

```
@@ -44,7 +44,10 @@ export interface CiPipelineAlertRequest {
   readonly environment?: string | null;
   readonly service?: string | null;
   readonly release_train_id?: string | null;
+  readonly release_operation_key?: string | null;
   readonly contributor_github_logins?: string[];
+  readonly contributor_evidence?:
+    'release-bus-operation' | 'manual-pr' | 'manual-range' | null;
```

ESLint

[error] Replace `'release-bus-operation'·|·'manual-pr'·|·'manual-range'` with `|·'release-bus-operation'⏎····|·'manual-pr'⏎····|·'manual-range'⏎···`

(prettier/prettier)